src/GraphQL.ApiTests/GraphQL.MicrosoftDI.approved.txt

```diff
@@ -1,5 +1,10 @@
 namespace GraphQL.MicrosoftDI
 {
+    public class AutoCreatingServiceProvider : System.IServiceProvider
```

Comment:
ActivatorUtilitiesServiceProvider 🤔  ?

Reply:
AutoCreatingServiceProvider doesn’t sound that great but ActivatorUtilitiesServiceProvider doesn’t even say what it does. Maybe SelfActivatingServiceProvider.

Reply:
SelfActivatingServiceProvider sounds a bit better than AutoCreatingServiceProvider.

Reply:
Maybe `GetOrCreateServiceProvider`

Reply:
Any name will fit, let's go on.